contrib/start_qemu.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+p=$(dirname $0)
+
+qemu-system-x86_64 \
+    -d 'cpu_reset' \
+    -enable-kvm \
+    -s \
+    -nographic \
+    -virtfs local,id=feeds,path=${p}/../openwrt/bin/packages/x86_64/,security_model=mapped-xattr,mount_tag=feeds \
+    -virtfs local,id=kmods,path=${p}/../openwrt/bin/targets/x86/64/packages/,security_model=mapped-xattr,mount_tag=kmods \
+    -netdev user,id=wan \
+    -device virtio-net-pci,netdev=wan,addr=0x06,id=nic1 \
+    -netdev user,id=lan,hostfwd=tcp::6080-192.168.1.1:80,hostfwd=tcp::2222-192.168.1.1:22,net=192.168.1.100/24 \
```

Comment:
This works well for the config mode, but not for when the normal runmode.

Reply:
Of course. This could be resolved later. :-)